src/Form/FormField.js

```diff
@@ -123,8 +123,11 @@ const SecondaryText = createStyledComponent('span', styles.secondaryText);
 const Caption = createStyledComponent('div', styles.caption);
 
 /**
+ * Mineral UI allows your app to highly customize the appearance of form inputs.
+ *
  * The FormField component wraps an input with an accessible label and displays
  * optional secondary text or a message.
+ * Styling will be applied if the wrapped element is a [TextField](../text-field).
```

Comment:
Please remove this line.  Styling will be applied to other mineral input components too, once they exist.  I'd rather just not call this out.

Reply:
👍